src/redux/actionConstant.js

```diff
@@ -0,0 +1,4 @@
+const GET_COMPANY = 'GET_COMPANY';
+export const GET_COMPANIES_DB = 'GET_COMPANIES_DB';
+export const GET_CHART_BY_FILTER = 'GET_CHART_BY_FILTER';
+export default GET_COMPANY;
```

Comment:
shouldn't export default, causes import to not be one line

Reply:
you can import default and not default in oneline
import {not default}, default1, default2.... from "../file"